### PR TITLE
Improve installation instructions

### PR DESCRIPTION
**Pull request by michaelrkn**

### CODE REVIEW

File: source/getting-started/index.md [hunk 1]
```diff
@@ -3,70 +3,66 @@ through our [command line build tool ember-cli](http://www.ember-cli.com/). The
...
+To verify that your installation was successful, run:
 
-Once you've got node.js installed, run the `node --version` command again to verify your install.
+```bash
+ember -v
+```
 
-Git comes with Mac OS X and most Linux distributions.
-Windows users can [download and run the installer](http://git-scm.com/download/win).
+If you get back something like `version: 0.2.0`, you're ready to go.
```
Comment (locks) [inline source/getting-started/index.md]: you could go slightly more modern here, like `1.13.0` :P

**Comment by locks** [comment]
LGMT apart from a nitpick.

**Comment by michaelrkn** [comment]
Nice catch. I've updated the PR. If it all looks good, can you merge please? Thank you!

**Comment by locks** [comment]
:boom: :hammer:

**Pull request merged**